modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -16584,6 +16586,7 @@ sibling=Sibling
 siblings=Siblings
 sic-code=SIC Code
 side-panel=Side Panel
+side-panel-title-help=Filling this field could turn out in a duplicated title if the selected URL has already a title for side panel.
```

Comment:
[nit] How about this wordsmithing:
`Filling this field could result in a duplicated title, if implementation on the URL already contains a title.`